src/main/kotlin/uk/gov/justice/digital/hmpps/nomisvisitsmappingservice/casenotes/CaseNotesMappingResource.kt

```diff
@@ -477,6 +478,28 @@ class CaseNotesMappingResource(private val mappingService: CaseNoteMappingServic
     @RequestParam(name = "onlyMigrated", required = false, defaultValue = "false") onlyMigrated: Boolean,
   ) = mappingService.deleteMappings(onlyMigrated)
 
+  @PutMapping("/merge/from/{oldOffenderNo}/to/{newOffenderNo}")
+  @Operation(
+    summary = "Replaces all occurrences of the 'from' id with the 'to' id in the mapping table",
+    description = "Used for update after a prisoner number merge. Requires role ROLE_NOMIS_CASENOTES",
+    responses = [
+      ApiResponse(responseCode = "200", description = "Replacement made, or not present in table"),
```

Comment:
seems a bit weird to return a `200` without a body?

Reply:
I thought that was ok for a put. Ill see what other endpoints are doing

Reply:
Ill give the function a block body